docs/framework/backbone.md

```diff
@@ -0,0 +1,306 @@
+# Backbone Reference
+
+!!! failure "Deprecated"
+    CiviCRM no longer recommends using Backbone. This page is here primarily for archival purposes.
+
+## Background
+
+Backbone is a model-view (MV) framework for Javascript which follows a
+minimalist, plugin-oriented architecture: at its core, Backbone defines
+a minimal set of classes and utilities for building user-interfaces with
+object-oriented Javascript. Like the Drupal and jQuery communities, the
+Backbone community has a wide range of plugins to address missing
+functionality.
+
+## Examples
+
+Backbone is currently used in the following parts of CiviCRM:
+
+-   Profile Designer (v4.3+): Drag/drop interface for creating
+    profile forms. (At time of writing, usable with CiviSurvey)
+-   CiviVolunteer Extension (v4.4+): Drag/drop interface for managing
```

Comment:
@seanmadsen it might be worth just checking in with Jamie about whether its still in use in the CiviVolunteer and CiviHR or if they have gone angular / bootstrap